src/components/member-page/email-subscription/subscription-options.js

```diff
@@ -175,16 +181,26 @@ const SubscriptionOptions = ({ jwt, userID, setUserData }) => {
 function mapStateToProps(state) {
   const jwt = _.get(state, [reduxStateFields.auth, 'accessToken'])
   const userID = _.get(state, [reduxStateFields.auth, 'userInfo', 'user_id'])
+  const readPreference = _.get(
+    state,
+    [reduxStateFields.user, 'readPreference'],
+    []
+  )
+  const maillist = _.get(state, [reduxStateFields.user, 'maillist'], [])
   return {
     jwt,
     userID,
+    readPreference,
+    maillist,
   }
 }
 
 SubscriptionOptions.propTypes = {
   jwt: PropTypes.string,
   userID: PropTypes.number,
   setUserData: PropTypes.func.isRequired,
+  readPreference: PropTypes.array,
```

Comment:
It seems like this page would not use `readPreference` value.
Could we remove it for now ?

Reply:
WE NEED THIS

Reply:
Is it because if we give null parameter to set user data api, it would clear user's readPreference value ?